tests/unit_tests/utils/dataframe_operations.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+import pytest
+from exasol_transformers_extension.utils import dataframe_operations
+
+
+sample_arr = [0, 1, 2, 2, 3, 3, 3, 4, 4, 4, 4, 5, 5, 5, 5, 5]
+sample_df = pd.DataFrame({
+    'A': sample_arr,
+    'B': sorted(sample_arr, reverse=True),
+    'C': list(range(len(sample_arr))),
+    'D': sample_arr
+})
+
+
+@pytest.mark.parametrize("columns, expected", [
+    (['A'], [[0], [1], [2], [3], [4], [5]]),
```

Comment:
it is helpful, if you can give your parameterization name, I usually, add a dummy str input , which describes what each single parameterization tests